src/wx/viewers.cpp

```diff
@@ -849,7 +875,7 @@ class MemViewer : public MemViewerBase {
         while (len > 0) {
             memoryMap m = map[addr >> 24];
             uint32_t off = addr & m.mask;
-            uint32_t wlen = (off + len) > m.mask ? m.mask + 1 - off : len;
+            int wlen = (off + len) > m.mask ? m.mask + 1 - off : len;
```

Comment:
Here we are losing type specificity, this is bad, better to find out why `uint32_t` does not work.

Reply:
This is a little bit tricky. The next line (855):
```
if (wlen < 0)
```
for the original code, this would never enter `if`. But there was a `return; // FIXME: give error`.  I would like to understand what this is about, but maybe it is not the time.

Reply:
In that case that's fine.